frontend/test/favorites/favorites_page_test.dart

```diff
@@ -0,0 +1,174 @@
+import 'package:ehrenamtskarte/configuration/configuration.dart';
+import 'package:ehrenamtskarte/favorites/favorites_model.dart';
+import 'package:ehrenamtskarte/favorites/favorites_page.dart';
+import 'package:ehrenamtskarte/l10n/translations.g.dart';
+import 'package:ehrenamtskarte/store_widgets/accepting_store_summary.dart';
+import 'package:ehrenamtskarte/store_widgets/detail/detail_app_bar.dart';
+import 'package:ehrenamtskarte/store_widgets/removed_store_content.dart';
+import 'package:ehrenamtskarte/store_widgets/removed_store_summary.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:graphql_flutter/graphql_flutter.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:provider/provider.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+import '../utils/fakes.dart';
+import '../utils/mocks.dart';
+
+void main() {
+  group('Favorites page', () {
+    setUpAll(() {
+      registerFallbackValue(FakeQueryOptions());
+      registerFallbackValue(FakeRoute());
+    });
+
+    final mockClient = MockGraphQLClient();
+    final mockObserver = MockNavigatorObserver();
+
+    // Create a wrapper widget for FavoritesPage to provide necessary context
+    Widget createTestWidget() {
+      return Configuration(
+        mapStyleUrl: 'dummy',
+        graphqlUrl: 'dummy',
+        projectId: 'dummy',
+        showDevSettings: false,
+        child: ChangeNotifierProvider<FavoritesModel>(
+          create: (_) => FavoritesModel()..initialize(),
+          child: MaterialApp(
+            navigatorObservers: [mockObserver],
+            home: TranslationProvider(
+              child: GraphQLProvider(
+                client: ValueNotifier(mockClient),
+                child: const FavoritesPage(),
+              ),
+            ),
+          ),
+        ),
+      );
+    }
+
+    testWidgets('Favorite store is available', (WidgetTester tester) async {
```

Comment:
```suggestion
    testWidgets('shows store in favorites if the store is available', (WidgetTester tester) async {
```
i prefer to use a more descriptive descriptions to get an impression what will be tested